CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# 1.0.1
+
+This is a major release that:
+
+1. Modifies the `lido_terra_hub` contract heavily;
+2. Introduces the new `lido_terra_token_stluna` token contract;
+3. Introduces the new `lido_terra_validators_registry` contract;
+4. Introduces the new `lido_terra_rewards_dispatcher` contract;
+5. Changes the reward distribution logic and applies a configurable fee to the rewards;
+6. Removes the option to pick a validator manually: now a validator is picked from an approved list in a way to make the delegation distribution more even.
```

Comment:
>now a validator is picked from an approved

this can be misleading, we do not pick exactly one validator, we try to distribute delegated amount equally, as documentation says https://lidofinance.github.io/terra-docs/contracts/hub#bond

Reply:
fixed